uizacoresdk/src/main/java/uizacoresdk/view/rl/video/UZPlayerManager.java

```diff
@@ -987,4 +994,8 @@ protected String getPixelAspectRatioString(float pixelAspectRatio) {
     public void addAdPlayerCallback(UZAdPlayerCallback uzAdPlayerCallback){
         this.uzAdPlayerCallback = uzAdPlayerCallback;
     }
+
+    private long calculateLiveStreamLatencyInMs(long startTimeUs) {
+        return System.currentTimeMillis() - startTimeUs / 1000;
```

Comment:
Should convert to `second`

Reply:
I think ms is OK because we will have sub second live streams.

Reply:
@deathemperor , agreed. However, should we use `second` unit in UI `start_for_nerds`?

Reply:
that's my point, using ms for everything including `stats for nerds`